atomic_reactor/plugins/pre_resolve_composes.py

```diff
@@ -354,21 +354,21 @@ def forward_composes(self):
                 repos.extend(noarch_repos)
 
         for arch, repofiles in repos_by_arch.items():
-            override_build_kwarg(self.workflow, 'yum_repourls', repofiles, arch)
+            override_build_kwarg(self.workflow, 'yum_repourls', repofiles, arch)  # OSBS2 TBD
```

Comment:
Explain those TBD here, why?

Reply:
Same as above for 'pre_pin_operator_digest.py'